api/lib/users.js

```diff
@@ -0,0 +1,45 @@
+var AWS = require("aws-sdk");
+var docClient = new AWS.DynamoDB.DocumentClient();
+
+module.exports ={
+	getUsers : function(event){
+		var datetime = new Date().getTime().toString();
+    var params = {};
+    params.TableName = "boilerplate-users-dev";
+    params.Item = {
+        "id": event.id,
+        "username": event.username,
+        
+    };
+
+    console.log(params.Item);
+
+    var pfunc = function(err, data) {
```

Comment:
What does this function do? IMO the name of the function should reflect the functionality which is missing here. 

And the code needs to be formatted before submitting the PR.